common/Driver.cs

```diff
@@ -0,0 +1,23 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace Hemz_QA_Mars.common
+{
+    public class Driver
+    {
+        public static IWebDriver driver;
+
+        public void Initialize()
+        {
```

Comment:
this is good! But can you please make use of hooks as well see https://docs.specflow.org/projects/specflow/en/latest/Bindings/Hooks.html


https://codebun.com/how-to-implement-hooks-in-specflow-c/

Reply:
Added Hooks and called it in before scenario (refer below)
https://github.com/Godzeye99/Mars_QA_repo/blob/onboarding_task/Hooks/Hooks.cs#L27